applications/StructuralMechanicsApplication/custom_elements/updated_lagrangian.h

```diff
@@ -77,6 +89,14 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) UpdatedLagrangian
     UpdatedLagrangian(IndexType NewId, GeometryType::Pointer pGeometry);
     UpdatedLagrangian(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties);
 
+    // Copy constructor
```

Comment:
you need this?

Reply:
It is supposed to be defined in all elements...